examples/tutorials/Exploring_Quantum_Chemistry_with_GDB1k.ipynb

```diff
@@ -79,7 +79,7 @@
     "id": "AssRCAgB2UJi"
```

Comment:
<p>why have you removed - <code>transformation.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>